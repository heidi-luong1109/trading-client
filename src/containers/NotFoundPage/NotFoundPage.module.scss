@import "../../assets/scss/_variables.scss";
@import "../../assets/scss/_breakpoints.scss";

.not-found-page {
  display: block;
  margin: 20px 0;
  
  &__content {
    width: 100%;
  }

  &__sidebar {
    width: 100%;
  }
}


@include breakpoint-lg {
  .not-found-page {
    display: flex;

    &__content {
      lost-column: 10/12;
    }

    &__sidebar {
      lost-column: 2/12; 
    }
  }
}
