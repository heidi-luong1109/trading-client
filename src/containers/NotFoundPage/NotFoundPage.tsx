import * as React from "react";
import { Helmet } from "react-helmet";
import { Sidebar } from "../../components/Sidebar";
import { Layout } from "../../components/Layout";
import s from "./NotFoundPage.module.scss";

const NotFoundPage = () => (
  <Layout>
    <Helmet>
      <title>Bitmex Client - Page not found</title>
    </Helmet>
    <div className={s["not-found-page"]}>
      <div className={s["not-found-page__sidebar"]}>
        <Sidebar />
      </div>
      <div className={s["not-found-page__content"]}>
        404 - Page not found!
      </div>
    </div>
  </Layout>
);

export default NotFoundPage;
