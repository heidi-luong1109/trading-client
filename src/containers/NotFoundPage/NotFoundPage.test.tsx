import "jsdom-global/register";
import React from "react";
import * as renderer from "react-test-renderer";
import NotFoundPage from "./NotFoundPage";

describe("NotFoundPage", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<NotFoundPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
