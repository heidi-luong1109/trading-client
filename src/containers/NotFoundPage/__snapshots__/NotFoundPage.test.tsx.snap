// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundPage renders correctly 1`] = `
<div>
  <div>
    <div>
      <div>
        <div>
          <h2>
            Bitmex Client
          </h2>
          <h2>
            v0.0.1
          </h2>
        </div>
        <div>
          <div>
            Select symbol:
          </div>
          <span
            className="bp3-popover-wrapper"
          >
            <span
              className="bp3-popover-target"
              onClick={[Function]}
            >
              <div
                className=""
                onKeyDown={[Function]}
              >
                <button
                  className="bp3-button"
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  type="button"
                >
                  <span
                    className="bp3-button-text"
                  >
                    XBTUSD
                  </span>
                  <span
                    className="bp3-icon bp3-icon-caret-down"
                    icon="caret-down"
                  >
                    <svg
                      data-icon="caret-down"
                      height={16}
                      viewBox="0 0 16 16"
                      width={16}
                    >
                      <desc>
                        caret-down
                      </desc>
                      <path
                        d="M12 6.5c0-.28-.22-.5-.5-.5h-7a.495.495 0 00-.37.83l3.5 4c.09.1.22.17.37.17s.28-.07.37-.17l3.5-4c.08-.09.13-.2.13-.33z"
                        fillRule="evenodd"
                      />
                    </svg>
                  </span>
                </button>
              </div>
            </span>
          </span>
        </div>
      </div>
    </div>
    <div>
      404 - Page not found!
    </div>
  </div>
</div>
`;
